src/net.cpp

```diff
@@ -1648,6 +1648,9 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
 
             if (add_fixed_seeds_now) {
                 std::vector<CAddress> seed_addrs{ConvertSeeds(Params().FixedSeeds())};
+                // We will not make outgoing connections to peers that are unreachable
+                // (e.g. because of -onlynet configuration).
+                // Therefore, it makes no sense to add them to addrman in the first place.
```

Comment:
Slight rewording nit

```suggestion
                // Therefore, we do not add them to addrman in the first place.
```

Reply:
or: `, so don't add them to addrman.`